src/main/java/com/gluonhq/substrate/target/AndroidTargetConfiguration.java

```diff
@@ -128,6 +128,9 @@ public boolean link() throws IOException, InterruptedException {
         if (ndk == null) throw new IOException ("Can't find an Android NDK on your system. Set the environment property ANDROID_NDK");
         if (clang == null) throw new IOException ("You specified an android ndk, but it doesn't contain "+ndk+"/toolchains/llvm/prebuilt/linux-x86_64/bin/clang");
         if (sdk == null) throw new IOException ("Can't find an Android SDK on your system. Set the environment property ANDROID_SDK");
+
+        compileAdditionalSources();
```

Comment:
why is this done (again) now in `link`?

Reply:
this is mainly done for Substrate development. In many cases, the launcher.c class is modified. If this is only compiled at compile time, it requires very long cycles.
But I'm ok removing it from the code, if nobody else is working on this.

Reply:
I usually have the same issue on iOS. I need to call to `compileAdditionalSources`, but not the rest of the compile task. 
Maybe we need a development flag, and when set, call this method also from `AbstractTargetConfiguration::link`?

Reply:
or we can just move it to link in general (for ios and android at least). Or have both.
The compile time for these additional resources is very small compared to the AOT compile time.

Reply:
Probably the latter (doing it as well from link) won't hurt in any case.

Reply:
... which is exactly what we have here :)

Reply:
yes, but only for Android, so maybe it should be for all platforms, in `AbstractTargetConfiguration`.